ui/src/components/credentials/IssueCredentialForm.tsx

```diff
@@ -213,182 +215,266 @@ export function IssueCredentialForm({
     [env, fetchJsonSchema, initialValues.schemaID, messageAPI]
   );
 
+  const computeFormObjectInitialValues = (
+    objectAttribute: ObjectAttribute,
+    initialValues: Record<string, unknown>
+  ): Record<string, unknown> | undefined => {
+    return objectAttribute.schema.attributes?.reduce(
+      (acc: Record<string, unknown>, curr: Attribute): Record<string, unknown> => {
+        switch (curr.type) {
+          case "boolean": {
+            const parsedConst = z.boolean().safeParse(curr.schema.const);
+            const parsedDefault = z.boolean().safeParse(curr.schema.default);
+            const constValue = parsedConst.success ? parsedConst.data : undefined;
+            const defaultValue = parsedDefault.success ? parsedDefault.data : undefined;
+            const value = constValue !== undefined ? constValue : defaultValue;
+            return { ...acc, [curr.name]: value };
+          }
+          case "integer":
+          case "number": {
+            const parsedConst = z.number().safeParse(curr.schema.const);
+            const parsedDefault = z.number().safeParse(curr.schema.default);
+            const constValue = parsedConst.success ? parsedConst.data : undefined;
+            const defaultValue = parsedDefault.success ? parsedDefault.data : undefined;
+            const value = constValue !== undefined ? constValue : defaultValue;
+            return { ...acc, [curr.name]: value };
+          }
+          case "string": {
+            const parsedConst = z.string().safeParse(curr.schema.const);
+            const parsedDefault = z.string().safeParse(curr.schema.default);
+            const constValue = parsedConst.success ? parsedConst.data : undefined;
+            const defaultValue = parsedDefault.success ? parsedDefault.data : undefined;
+            const value = constValue !== undefined ? constValue : defaultValue;
+            if (value === undefined) {
+              return acc;
+            }
+            switch (curr.schema.format) {
+              case "date":
+              case "date-time": {
+                return { ...acc, [curr.name]: dayjs(value) };
+              }
+              case "time": {
+                return { ...acc, [curr.name]: dayjs(`1978-01-29T${value}`) };
```

Comment:
What is this `dayjs(`1978-01-29T${value}`)`? 🤣

Reply:
My birthday

Reply:
🤣 but i mean, what's the purpose of this random date to concatenate the time?

Reply:
Changed to `1970-01-01` as agreed.